### PR TITLE
Added code to upload bom folder

### PR DESCRIPTION
**Pull request by bajajyukta7**
## Problem
For BPaaS, we want customer to use software download script to
1. upload the bom folder to the storage account
2.  print the bom folder path in storage account as output

## Solution
Added code to solve the problem statement

## Tests
Verified locally

## Notes
<Additional comments for the PR>

### CODE REVIEW

**Review by hdamecharla — CHANGES_REQUESTED** [review]
Hi @yuktabajaj/ @bajajyukta7, 
Kindly review line 85 of deploy/ansible/roles-sap/0.1-bom-validator/tasks/bom_validator.yaml, please? This needs to match the path of the folder from where we would upload the BOMs and it currently references a static deployment path.

**Comment by bajajyukta7** [comment]
> Hi @yuktabajaj / @bajajyukta7, Kindly review line 85 of deploy/ansible/roles-sap/0.1-bom-validator/tasks/bom_validator.yaml, please? This needs to match the path of the folder from where we would upload the BOMs and it currently references a static deployment path.

Hi @hdamecharla, I have changed the path from static to dynamic var. Kindly review.

File: deploy/ansible/roles-sap/0.1-bom-validator/tasks/main.yaml [hunk 1]
```diff
@@ -66,6 +66,9 @@
     bom_name:                          "{{ bom_base_name }}"
 # -------------------------------------+---------------------------------------8
 
+- name: Bom folder path
+  debug:
```
Comment (KimForss) [inline deploy/ansible/roles-sap/0.1-bom-validator/tasks/main.yaml:70]:
```suggestion
  ansible.builtin.debug:
```

File: deploy/ansible/roles-sap/0.1-bom-validator/tasks/main.yaml [hunk 2]
```diff
@@ -66,6 +66,9 @@
...
+    msg: "{{ sapbits_location_base_path }}/boms/{{ bom_base_name }}"
```
Comment (KimForss) [inline deploy/ansible/roles-sap/0.1-bom-validator/tasks/main.yaml:71]:
```suggestion
    msg:                           "{{ sapbits_location_base_path }}/boms/{{ bom_base_name }}"
```

**Review by KimForss — CHANGES_REQUESTED** [review]

File: deploy/ansible/roles-sap/0.1-bom-validator/tasks/main.yaml (hunk 1 above)
Comment (yuktabajaj) [reply deploy/ansible/roles-sap/0.1-bom-validator/tasks/main.yaml]: > 

Resolver in latest commit

File: deploy/ansible/roles-sap/0.1-bom-validator/tasks/main.yaml (hunk 2 above)
Comment (yuktabajaj) [reply deploy/ansible/roles-sap/0.1-bom-validator/tasks/main.yaml]: > 

resolved in commit b50c03666a268e0527a05b30033c2ef206b31899

**Review by KimForss — APPROVED** [review]

**Review by KimForss — APPROVED** [review]

**Pull request merged**